packages/compute/src/vm.js

```diff
@@ -579,4 +637,24 @@ VM.prototype.request = function(reqOpts, callback) {
   });
 };
 
+VM.prototype.execAfterOperation_ = function(callback) {
```

Comment:
It doesn't have to be. And it should really go on `Compute`, as we might want this from many methods. And *really*, it should be exposed to the user, but not like that ^, like this:

```js
vm.stop({ operation: false }, function(err, results) {
  // operation completed
})
```

... but that's a problem for a different day.